src/Core/Usecase/HXL/SearchHXLLicense.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rowasc
+ * Date: 5/7/18
+ * Time: 2:37 PM
+ */
+
+namespace Ushahidi\Core\Usecase\HXL;
+
+use Ushahidi\Core\Usecase\SearchUsecase;
+
+class SearchHXLLicense extends SearchUsecase
```

Comment:
How does this differ from SearchUsecase? If it changes interact can you make sure its clear what changed?

Reply:
removed. Not necessary